### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Vgonma**
# Linters

- Setup linters for HTML, CSS, and JavaScript
- # Sekeleton
- Set up index.html and index.css for the markup and style.

# Functionality

- Split functions into modules.
- Add import statements.
- Add javascript functionality to change pages.
- Add Book list functionality to save and load books.
- Add New book functionality to add books to the collection.

# Readme

- Update README.md to look professional

### CODE REVIEW

File: modules/bookList.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import * as SL from './saveLoad.js';
+
+function addRemoveOnClick(button, collection) {
```
Comment (FrederickMih) [inline modules/bookList.js]: - [x] Based on the requirements of this project, we are supposed to define our functions using ES6 arrow syntax. No `function` keyword is allowed for this Milestone @Vgonma :+1: .

File: modules/bookList.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  button.addEventListener('click', () => {
+    const booksContainer = button.parentElement.parentElement;
+    const book = button.parentElement;
+    collection.removeBook(Array.prototype.indexOf.call(booksContainer.children, book));
+    booksContainer.removeChild(book);
+    SL.saveLocalStorage('collection', collection.collection);
+  });
+}
+
+export default function displayColection(DOMElement, collection) {
```
Comment (FrederickMih) [inline modules/bookList.js]: - [x] Arrow syntax only :+1:

File: modules/navInteraction.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export default function addTransitionToMenu(navList) {
```
Comment (FrederickMih) [inline modules/navInteraction.js]: - [x] Arrow syntax only :arrow_up:

File: modules/saveLoad.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+export function saveLocalStorage(key, object) {
```
Comment (FrederickMih) [inline modules/saveLoad.js:1]: - [x] Kindly check the comment up :arrow_up: Arrow syntax only

File: modules/saveLoad.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+  if (!key || !object) { return; }
+  localStorage.setItem(key, JSON.stringify(object));
+}
+
+export function loadLocalStorage(key, object) {
```
Comment (FrederickMih) [inline modules/saveLoad.js:6]: - [x] :arrow_up:

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand:  @Vgonma , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there

Screenshot from 2022-10-20 09-38-51
!




## Highlights :green_circle: 

- Great job for dividing your code into reasonable modules :+1: 
- App is still working as expected :+1: 
- Fantastic commit history :+1: 
- Descriptive PR :+1: 
- No Linter error :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @Vgonma, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**